construct_predictions.py

```diff
@@ -50,7 +50,7 @@ class ConstructPredictionsPrimitive(transformer.TransformerPrimitiveBase[Inputs,
             'id': '8d38b340-f83f-4877-baaa-162f8e551736',
             'version': '0.3.0',
             'name': "Construct pipeline predictions output",
-            'python_path': 'd3m.primitives.data_transformation.construct_predictions.DataFrameCommon',
+            'python_path': 'd3m.primitives.data_transformation.construct_predictions.Common',
```

Comment:
do we actually use this file?

Reply:
I'll have to double check this. The `dockerfile` calls out this being added, and I thought I needed to make this update to resolve an error at one point, but now that I am testing again, I'm not so sure.

Reply:
This is being added to the container by the `dockerfile`, but that may not be necessary. At least a couple of the tests run without the `ADD` from the `dockerfile`. I can do some more comprehensive testing and delete this file and update the `dockerfile` accordingly if it is no longer needed. Do you know why it was set up this way to begin with?